### ISSUE TITLE
AttributeError: class img2net has no attribute 'calc'

### ISSUE THREAD

**Issue by gzliuqingyun** [DavidBreuer/img2net#1]
```
Traceback (most recent call last):
  File "run_img2net.py", line 70, in on_button1_clicked
    img2net.calc.calc(self,gtk,dir_input,gridtype,nullmodel,R,lz,dx,dy,dz,pbx,pby,pbz,vax,vay,cores)
AttributeError: class img2net has no attribute 'calc'
```
As I list img2net's methods/attributes, the result below:
```
>>> dir(img2net)
['__builtins__', '__doc__', '__file__', '__name__', '__package__', '__path__', '__version__']
```

**Comment by DavidBreuer** [comment]
Hi @gzliuqingyun, right, thanks for your feedback! I chose a class name unwisely which caused a conflict with the package name. This and another issue has been resolved now. Can you confirm that it's working now? Thanks and best.

PS: Calling dir(img2net) shows the 'calc' module now, but only after importing both 'import img2net' and 'import img2net.calc'.

**Comment by DavidBreuer** [comment]
Closed by 5e5c544899a1107e9ab8d6f6be721890bd967e52.
